piksi_tools/console/settings.yaml

```diff
@@ -2199,3 +2199,202 @@
   Description: (ALPHA) Maximum possible linear acceleration of vehicle.
   Notes: Has no effect unless the vehicle dynamics filter is active.
 
+- group: ins
+  name: output_mode
+  expert: false
+  type: enum
+  units: N/A
+  default value: 'disabled'
+  readonly: false
+  enumerated possible values: Disabled, Loosely Coupled, LC + GNSS
+  Description: Determines output mode of the inertial navigation outputs. 
+  Notes: Disabled configures device to output GNSS only solutions.
+         Loosely Coupled configures one loosely coupled blended GNSS and inertial solution per epoch.
+         LC + GNSS configures a testing mode that simultanously outputs both GNSS only and INS solution solution messages each epoch.
+         In LC + GNSS mode, there will two sets of solution messages with indentical times. The flags indicate which messages
+         are GNSS only message and which come from the INS system.
+
+- group: ins
+  name: odometry_noise_1
+  expert: true
+  type: double
+  units: m/s
+  default value: '0.28'
+  readonly: false
+  Description: Noise parameter for odometry source 1
+
+- group: ins
+  name: odometry_noise_2
+  expert: true
+  type: double
+  units: m/s
+  default value: '0.28'
+  readonly: false
+  Description: Noise parameter for odometry source 2
+
+- group: ins
+  name: odometry_noise_3
+  expert: true
+  type: double
+  units: m/s
+  default value: '0.28'
+  readonly: false
+  Description: Noise parameter for odometry source 3
+
+- group: ins
+  name: odometry_noise_4
+  expert: true
+  type: double
+  units: m/s
+  default value: '0.28'
+  readonly: false
+  Description: Noise parameter for odometry source 4
+
+- group: ins
+  name: vehicle_frame_roll
+  expert: false
+  type: double
+  units: degrees
+  default value: '0'
+  readonly: false
+  Description: Roll angle representing rotation from vehicle frame to device frame.  
```

Comment:
I thought we were moving away from roll, pitch, and yaw and instead were going to describe the clockwise/counterclockwise rotations about an axix.

Reply:
I was hoping to do that, but I am just leaving well enough alone.  Forthcoming are user docs to explain these and provide a worksheet to configure.